docs/yum-run.md

```diff
@@ -3,38 +3,36 @@
 Percona Server for MySQL stores the data files in `/var/lib/mysql/` by
 default. The configuration file used to manage Percona Server for MySQL is the `/etc/my.cnf`.
 
-The following commands start, provide the server status, stop the server, and restart the server.
+## Manage the service
 
-!!! note
+You would use `systemctl` to manage system services and daemons in a Linux environment. It provides a systematic and unified interface for controlling the state of services and checking their status. With `systemctl`, you can start a service to initiate its operations, stop it to terminate its operations, restart it to refresh its state, and check the status to monitor its performance and health. This command is essential for system administrators who need precise control over service management tasks.
 
-    The RHEL distributions and derivatives come with [systemd](https:/freedesktop.org/wiki/Software/systemd/) as the default system and service manager so you can invoke all of the commands with `sytemctl` instead of `service`. Currently, both options are supported.
+The RHEL distributions and derivatives come with [systemd](https:/freedesktop.org/wiki/Software/systemd/) as the default system and service manager. 
 
-1. Percona Server for MySQL is not started automatically on the RHEL distributions and derivatives after installation. Start the server with the following command:
-	
-	```{.bash data-prompt="$"}
-	$ sudo service mysql start
-	```
+`systemctl` is a command-line utility that is used to control the systemd system and service manager. It's a primary tool for managing services on Linux distributions that use systemd.
 
-2. Review the service status with the following command:
+You can use either `sytemctl` or `service`. Currently, both options are supported.
 
-	```{.bash data-prompt="$"}
-	$ sudo service mysql status
-	```
+Percona Server for MySQL is not started automatically on the RHEL distributions and derivatives after installation. 
+
+The following command uses superuser privileges to start, check the status, stop, and restart the MySQL service using `systemctl`.
 
-3. Stop the service with the following command:
-	
 	```{.bash data-prompt="$"}
-	$ sudo service mysql stop
+	$ sudo systemctl start mysql
+	$ sudo systemctl status mysql
+	$ sudo systemctl stop mysql
+	$ sudo systemctl restart mysql
 	```
```

Comment:
This command is broken on the preview

Reply:
It works in Docker.

Reply:
It's broken on Render preview https://percona-server-docs-8-0-pr-296.onrender.com/yum-run.html#manage-the-service